templates/CMakeLists.txt.template

```diff
@@ -345,49 +370,53 @@
   #   Add custom commands to process ``.proto`` files to C++ using protoc and
   #   GRPC plugin::
   #
-  #     protobuf_generate_grpc_cpp [<ARGN>...]
+  #     protobuf_generate_grpc_cpp <FILE_LOCATION> <IMPORT_PATH>
   #
-  #   ``ARGN``
-  #     ``.proto`` files
+  #   ``FILE_LOCATION``
+  #     The relative path of the ``.proto`` file to the project root
+  #   ``IMPORT_PATH``
+  #     The proto import path that itself expected to be placed in
```

Comment:
I think it would be useful to provide a small example here.
is "IMPORT_PATH" the path under which other .proto files reference this file when they do `import "import/path/this.proto"`?